beacon-chain/rpc/aggregator/server_test.go

```diff
@@ -27,6 +28,13 @@ func init() {
 	params.OverrideBeaconConfig(params.MinimalSpecConfig())
 }
 
+// _pubKey is a helper to generate a well-formed public key.
+func _pubKey(i uint64) []byte {
```

Comment:
Please remove the underscore. Go naming convention is `mixedCase` and lower case functions are considered private.

https://golang.org/doc/effective_go.html#mixed-caps

Reply:
Reason I did that is because the test is in the `aggregator` package and I didn't want a function used only for testing and in a `_test` file to potentially be used by functions in the non-test package.  Will change it if you're okay with this (there's another one out there in another package I'll change as well).

Reply:
The compiler enforces that you can't use code from _test files in non test files so it is safe to do this.

Reply:
Cool; never knew that.  Updated.